### ISSUE TITLE
Strange behavior of the images after the tests are finished

### ISSUE THREAD

**Issue by Miroslav-Stopka** [shopsys/shopsys#488]
### What is happening
Demo images are being imported as a part of the data fixture during the build. When the web is displayed at this time - before I run the test, I see all the demo images.

After the tests, however, some images are missing - for example, the first slider image or the image of the product 22 "Sencor SLE 22F46DM4 HELLO KITTY.

### Expected result
There are no missing demo images after the tests are finished.

**Comment by Miroslav-Stopka** [comment]
It seems like the problem is caused by Http smoke tests.

**Comment by PetrHeinz** [comment]
Yes, HTTP smoke test try removing a entities (with ID 1 by default) using these routes:
```
  admin_administrator_delete     /admin/administrator/delete/{id}
  admin_advert_delete            /admin/advert/delete/{id}
  admin_article_delete           /admin/article/delete/{id}
  admin_availability_delete      /admin/product/availability/delete/{id}
  admin_brand_delete             /admin/brand/delete/{id}
  admin_category_delete          /admin/category/delete/{id}
  admin_currency_delete          /admin/currency/delete/{id}
  admin_customer_delete          /admin/customer/delete/{id}
  admin_flag_delete              /admin/product/flag/delete/{id}
  admin_newsletter_delete        /admin/newsletter/delete/{id}
  admin_order_delete             /admin/order/delete/{id}
  admin_orderstatus_delete       /admin/order-status/delete/{id}
  admin_parameter_delete         /admin/product/parameter/delete/{id}
  admin_payment_delete           /admin/payment/delete/{id}
  admin_pricinggroup_delete      /admin/pricing/group/delete/{id}
  admin_product_delete           /admin/product/delete/{id}
  admin_promocode_delete         /admin/promo-code/delete/{id}
  admin_slider_delete            /admin/slider/item/delete/{id}
  admin_transport_delete         /admin/transport/delete/{id}
  admin_unit_delete              /admin/product/unit/delete/{id}
  admin_vat_delete               /admin/vat/delete/{id}
```
The requests are in a transaction which is rolled back, but the changes to files on the disk are not reverted. This problem did not occur before #476 as there were no images in the test DB.

I would recommend fixing this by having a separate `content` directory for tests, maybe by something like 95cf84622.

**Comment by Miroslav-Stopka** [comment]
Solved in #547
